sdks/python/tox.ini

```diff
@@ -17,7 +17,7 @@
 
 [tox]
 # new environments will be excluded by default unless explicitly added to envlist.
-envlist = py38,py39,py310,py311,py38-{cloud,cython,docs,lint,mypy,cloudcoverage,dask},py39-{cloud,cython},py310-{cloud,cython,dask},py311-{cloud,cython,dask},whitespacelint
```

Comment:
this actually removes a test suite that covers `no cloud dependencies` variant.

A while ago we had plain suite, gcp suite , and cython suite. Later we said, why don't we get rid of one set of tests by having: `cython` + `no cloud` as one suite, and cloud as another suite. 

Loss of coverage would impact the scenario when one installs:

pip install apache-beam (without cloud/gcp extras). One could argue - should we even have this test suite or codepath? but that would be a separate conversation. For now, we could perhaps rename the `cython` suite into `noextras`.

Reply:
We already have a test suite https://github.com/apache/beam/blob/a7bf59436a664dca932c5001b750514dab2fa296/sdks/python/tox.ini#L72 that can be treated as no-extras. I will reuse it.